.github/workflows/conformance-runtime.yaml

```diff
@@ -188,8 +188,6 @@ jobs:
           # RuntimeAgentChaos Cilium agent removing leftover Cilium interfaces
           # RuntimeAgentChaos Connectivity over restarts Checking that during restart no traffic is dropped using Egress + Ingress Traffic
           # RuntimeAgentChaos Endpoint Endpoint recovery on restart
-          # RuntimeAgentChaos KVStore Delete event on KVStore with CIDR identities
-          # RuntimeAgentChaos KVStore Validate that delete events on KVStore do not release in use identities
```

Comment:
Aren't we losing coverage by removing these two tests?

Reply:
Despite not being clear in their names, these two tests were Consul-specific, with even a hardcoded Consul URL.